budgetportal/tests/helpers.py

```diff
@@ -45,3 +46,51 @@ def wait_until_text_in(self, selector, text):
         self.wait.until(
             EC.text_to_be_present_in_element((By.CSS_SELECTOR, selector), text)
         )
+
+    def _fixture_teardown(self):
```

Comment:
I want to come back to this later - not sure yet why this is needed. just hiding the file for now but keeping this comment

Reply:
Once wagtail is installed there is some internal issue with TranscationalTaseCases.
The fix is to flush DB using allow cascade=True option.

We override _fixture_teardown to make it.